src/test/java/pages/FileUploadPage.java

```diff
@@ -0,0 +1,59 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class FileUploadPage extends BasePage{
+    public final String FILE_UPLOAD_URL = "http://the-internet.herokuapp.com/upload";
+    public final String FILE_PATH = "C:\\Users\\dkwr2\\OneDrive\\Desktop\\test.txt";
```

Comment:
FILE_PATH should not be in Page class, it's Test data

Reply:
done